database.sql

```diff
@@ -0,0 +1,35 @@
+DROP TABLE IF EXISTS question_reponse;
```

Comment:
What is the purpose of this file at the root of the project that insert these questions in the database?